### ISSUE TITLE
Not able to build the mkdocs using mkdocs build command

### ISSUE THREAD

**Issue by 6vedant** [brospars/mkdocs-page-pdf#7]
I get following error

```
vj@VJs-MacBook-Pro techlib-source % mkdocs build
INFO     -  DeprecationWarning: There is no current event loop
              File "/opt/homebrew/lib/python3.10/site-packages/nest_asyncio.py", line 45, in _get_event_loop
                loop = events.get_event_loop_policy().get_event_loop()
              File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/events.py", line 666, in get_event_loop
                warnings.warn('There is no current event loop',
Traceback (most recent call last):
  File "/opt/homebrew/bin/mkdocs", line 8, in <module>
    sys.exit(cli())
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/opt/homebrew/lib/python3.10/site-packages/mkdocs/__main__.py", line 250, in build_command
    build.build(cfg, dirty=not clean)
  File "/opt/homebrew/lib/python3.10/site-packages/mkdocs/commands/build.py", line 270, in build
    config = config.plugins.run_event('config', config)
  File "/opt/homebrew/lib/python3.10/site-packages/mkdocs/plugins.py", line 520, in run_event
    result = method(item, **kwargs)
  File "/opt/homebrew/lib/python3.10/site-packages/mkdocs_page_pdf/plugin.py", line 60, in on_config
    onServe = config['dev_addr'].host+':'+str(config['dev_addr'].port) in config['site_url']
TypeError: argument of type 'NoneType' is not iterable
```

**Comment by brospars** [comment]
You need to set the `site_url` property (more info [here](https://www.mkdocs.org/user-guide/configuration/#site_url)). 

I should probably fix that since the default value is null.

**Comment by 6vedant** [comment]
`mkdocs build` works fine without this plugin. So it should work fine also with this plugin as well. Please help in fixing it as well. I will appreciate a lot.

**Comment by 6vedant** [comment]
if the `site_url `is null in the config, the default value should be `http://127.0.0.1:8000`

**Comment by brospars** [comment]
That what I meant. I'll fix it as soon as possible but PR are welcome 😉

**Comment by 6vedant** [comment]
https://github.com/brospars/mkdocs-page-pdf/pull/8
If you like the change let me know

**Comment by brospars** [comment]
Fixed by [d8f3ebd](https://github.com/brospars/mkdocs-page-pdf/commit/d8f3ebd697a3003d693fda77bc6b0c49ffce2790)
